src/app/buy-credits/buy-credits.component.scss

```diff
@@ -0,0 +1,61 @@
+@use '@angular/material' as mat;
+@import '../../abstract';
+
+#loggedInHeader {
+    background-color: green;
+    color: white;
+    margin: 0px;
+    padding: 10px 2rem;
+    display: flex;
+    flex-direction: row-reverse;
+
+    p {
+        margin: 0;
+    }
+}
+
+.mat-stepper-vertical {
+    margin-top: 8px;
+}
+
+.mat-form-field {
+    margin-top: 16px;
+}
+
+button[mat-button] {
+    background-color: mat.get-color-from-palette($donate-primary);
+    color: white;
+    box-shadow: $box-shadow-hard;
+}
+
+.tip-text {
+    margin-bottom: 0;
+}
+
+#giftAidOptIn {
+    margin-bottom: 20px;
+}
+
+.error {
+    color: mat.get-color-from-palette($donate-warn);
+    font-size: 1rem; // slightly smaller text
```

Comment:
Why?

Reply:
It looks nicer imo, otherwise it's the same size as the normal text.